src/workloads/execution/ClusteredCollection.yml

```diff
@@ -21,8 +21,15 @@ AutoRun:
       $eq:
       - standalone
       - standalone-all-feature-flags
+      - standalone-query-stats
+      - standalone-query-stats-small-rate-limit
       - replica
       - replica-all-feature-flags
+      - replica-query-stats
+      - atlas-like-replica.2022-10
```

Comment:
Why are we adding non-query stats workloads too? (`shard` and `atlas-like-replica.2022-10`)

Reply:
Oh sorry I never responded to this! This is because we use the non-query-stats version as the baseline performance, then compute the diff to see how we compare. That's what's plotted in all our graphs on the dashboard.